### PR TITLE
Pylint fix

### PR DESCRIPTION
**Pull request by domfournier**

### CODE REVIEW

File: octree_creation_app/utils.py [hunk 1]
```diff
@@ -220,12 +236,26 @@ def octree_2_treemesh(  # pylint: disable=too-many-locals
...
+        if (
+            getattr(mesh, f"{ax}_cell_size") is None
+            or getattr(mesh, f"{ax}_count") is None
+        ):
+            raise ValueError(f"Cell size in {ax} direction is not defined.")
+
+        n_cell_dim.append(getattr(mesh, f"{ax}_count"))
+        cell_sizes.append(
+            np.ones(getattr(mesh, f"{ax}_count")) * getattr(mesh, f"{ax}_cell_size")
+        )
```
Comment (sebhmg) [inline octree_creation_app/utils.py:258]: **suggestion:** more readable if assigning after the loop, while loop is dedicated to safety checks

```suggestion
    n_cell_dim = [mesh.u_count, mesh.v_count, mesh.w_count]
    cell_sizes = [
        np.ones(mesh.u_count) * mesh.u_cell_size,
        np.ones(mesh.v_count) * mesh.v_cell_size,
        np.ones(mesh.w_count) * mesh.w_cell_size
    ]
```

**Review by sebhmg — CHANGES_REQUESTED** [review]
see suggested change for code readability

File: octree_creation_app/utils.py (hunk 1 above)
Comment (sebhmg) [reply octree_creation_app/utils.py:258]: but that can be applied to develop. Still good to merge here

Comment (domfournier) [reply octree_creation_app/utils.py:258]: Unfortunetly, the linters aren't that smart and don't take into account the getattr for the checks...

**Pull request merged**